src/main/java/RaceController.java

```diff
@@ -1,14 +1,19 @@
 public class RaceController {
     private final RaceGame game;
-    private final int roundCount;
+    private final RoundCount roundCount;
 
     public RaceController(Cars cars, int roundCount) {
-        this.game = new RaceGame(cars);
+        this(new RaceGame(cars), new RoundCount(roundCount));
+    }
+
+    public RaceController(RaceGame game, RoundCount roundCount) {
+        this.game = game;
         this.roundCount = roundCount;
     }
 
     public void start() {
-        for (int round = 0; round < roundCount; round++) {
+        ResultView.printResultHeader();
+        for (int round = 0; round < roundCount.count(); round++) {
```

Comment:
```suggestion
        for(game.isRacing()) {
```
roundCount의 값을 꺼내지 말고 game에 메시지를 보내 위와 같이 구현하는 것은 어떨까?